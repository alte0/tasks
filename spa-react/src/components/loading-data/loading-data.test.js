import React from 'react';
import LoadingAata from "./loading-data";
import renderer from 'react-test-renderer';

it('LoadingData render', () => {
    const tree = renderer
        .create(<LoadingAata/>)
        .toJSON();

    expect(tree).toMatchSnapshot();
});
