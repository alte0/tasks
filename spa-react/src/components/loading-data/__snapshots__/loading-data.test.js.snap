// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingData render 1`] = `
<div
  className="loader-container"
>
  <div
    className="loader"
  >
    <div
      className="text"
    >
      Загрузка данных
    </div>
    <div
      className="dots"
    >
      <div
        className="dot"
      />
      <div
        className="dot"
      />
      <div
        className="dot"
      />
      <div
        className="dot"
      />
    </div>
  </div>
</div>
`;
